codespell_lib/data/dictionary.txt

```diff
@@ -23592,6 +23592,7 @@ eeeprom->EEPROM
 eeger->eager
 eegerly->eagerly
 eejus->aegis
+eerything->everything
```

Comment:
Unlikely, but it's possible that they meant "eerie thing".
```suggestion
eerything->eerie thing, everything,
```

Reply:
Nice, thanks a lot. Wasn't even aware that such a word `eerie` exists :-)

Updated with the last force push but placed `everything` first because this seems to be more likely.